knowledge_graphs/ai_script_analyzer.py

```
@@ -114,8 +124,8 @@ def analyze_script(self, script_path: str) -> AnalysisResult:
             self.context_manager_vars.clear()
 
             # Track processed nodes to avoid duplicates
-            self.processed_calls = set()
-            self.method_call_attributes = set()
+            self.processed_calls: set[int] = set()
```

Pylint (3.3.7)

[warning] Attribute 'processed_calls' defined outside __init__

(W0201)